CONTRIBUTING.md

```diff
@@ -0,0 +1,15 @@
+CONTRIBUTING.md
+
+1. find issue and fork
+
+2. make changes
+
+3. run tests
+- a) `cargo test`
+- b) `cargo clippy`
+- c) `cargo rustfmt`
+
+4. commit format
+- TODO: commit message formatting needed
```

Comment:
This repo hasn't had an official commit message policy. Perhaps we should adopt one.